courses/serializers.py

```diff
@@ -201,3 +202,38 @@ class Meta:
             'requirements',
             'sections',
         ]
+
+
+class UserSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = get_user_model()
+        fields = [
+            'username',
+            'first_name',
+            'last_name',
+        ]
+
+
+class UserDataSerializer(serializers.ModelSerializer):
+    user = UserSerializer(read_only=True)
+
+    class Meta:
+        model = UserData
+        fields = [
+            'email',
+            'phone',
+            'user'
```

Comment:
Don't think it matters too much in the end, since it's a one-to-one relationship, but to me having the user data nested within the user seems a bit more natural